### ISSUE TITLE
Conda detection / setup fails under Ubuntu 22.04

### ISSUE THREAD

**Issue by kfsone** [microsoft/windows-ai-studio#32]
Windows 11 -> Ubuntu 22.04. I've restarted/retried repeatedly, The requirements validation immediately claims there's no conda install. I clicked Setup WSL Environment which failed to solve the conda issue. I manually installed miniconda3, set CONDA_HOME to ~/miniconda3, and even after restarts, still no conda detection.

The problem appears to be that you are expecting /opt/miniconda when the current miniconda3 installer installs to ~/miniconda3 by default. Creating a symlink from /opt/miniconda -> ~/miniconda3 fixes this.

See also: https://github.com/microsoft/windows-ai-studio/issues/22

```
[2023-12-20T03:05:37.990Z] [INFO] Command registration.
[2023-12-20T03:05:37.991Z] [INFO] Validating environment on activation.
[2023-12-20T03:05:37.991Z] [INFO] Extenension: Invoking validateEnvironement for: All
Debug: validate-env[0]
   07:05:38.04 0 ExecuteAsync Started
Information: validate-env[0]
   07:05:38.26 0 IsWSLDetected Execution
Error: validate-env[0]
   07:05:38.61 0 Error: No LSB modules are available.

Information: validate-env[0]
   07:05:38.61 0 The default WSL distribution is Ubuntu 18.04 or greater.
Information: validate-env[0]
   07:05:38.61 0 IsNvidiaDiverAvailable Execution
Information: validate-env[0]
   07:05:38.91 0 IsNvidiaDiverAvailable : True
Information: validate-env[0]
   07:05:38.91 0 IsCudaRuntimeInstalled Execution
Information: validate-env[0]
   07:05:39.07 0 IsCudaRuntimeInstalled : True
Information: validate-env[0]
   07:05:39.07 0 IsCondaInstalled Execution
Information: validate-env[0]
   07:05:39.23 0 IsCondaInstalled : False
Debug: validate-env[0]
   07:05:39.23 0 ExecuteAsync Completed Elapsed:00:00:01.2025383
[2023-12-20T03:05:39.341Z] [INFO] Validation outcome prereqsmet: false
[2023-12-20T03:05:39.341Z] [INFO] Initialize WebView pane.
[2023-12-20T03:05:39.342Z] [INFO] Navigating view to requirements page.
[2023-12-20T03:05:39.805Z] [INFO] Navigation to loadPrerequisiteCheck was successful: true
[2023-12-20T03:05:39.805Z] [INFO] Loading View: loadPrerequisiteCheck
[2023-12-20T03:05:39.814Z] [INFO] Extenension: Invoking validateEnvironement for: conda
Debug: validate-env[0]
   07:05:39.87 0 ExecuteAsync Started
Information: validate-env[0]
   07:05:40.10 0 IsCondaInstalled Execution
Information: validate-env[0]
   07:05:40.46 0 IsCondaInstalled : False
Debug: validate-env[0]
   07:05:40.46 0 ExecuteAsync Completed Elapsed:00:00:00.6047748
[2023-12-20T03:05:40.481Z] [INFO] Extenension: Invoking validateEnvironement for: cuda-runtime
Debug: validate-env[0]
   07:05:40.59 0 ExecuteAsync Started
[2023-12-20T03:05:40.727Z] [INFO] telemetry event:activate_extension sent
[2023-12-20T03:05:40.727Z] [INFO] telemetry event:requirements_needed sent
Information: validate-env[0]
   07:05:40.88 0 IsCudaRuntimeInstalled Execution
Information: validate-env[0]
   07:05:41.14 0 IsCudaRuntimeInstalled : True
Debug: validate-env[0]
   07:05:41.14 0 ExecuteAsync Completed Elapsed:00:00:00.5682699
[2023-12-20T03:05:41.152Z] [INFO] Extenension: Invoking validateEnvironement for: nvidia-driver
Debug: validate-env[0]
   07:05:41.22 0 ExecuteAsync Started
Information: validate-env[0]
   07:05:41.37 0 IsNvidiaDiverAvailable Execution
Information: validate-env[0]
   07:05:41.81 0 IsNvidiaDiverAvailable : True
Debug: validate-env[0]
   07:05:41.81 0 ExecuteAsync Completed Elapsed:00:00:00.5973269
[2023-12-20T03:05:41.819Z] [INFO] Extenension: Invoking validateEnvironement for: wsl-detected
Debug: validate-env[0]
   07:05:41.86 0 ExecuteAsync Started
Information: validate-env[0]
   07:05:41.96 0 IsWSLDetected Execution
Debug: validate-env[0]
   07:05:42.05 0 ExecuteAsync Completed Elapsed:00:00:00.1940328
[2023-12-20T03:05:42.061Z] [INFO] Extenension: Invoking validateEnvironement for: wsl-distro-ubuntu1804OrGreater
Debug: validate-env[0]
   07:05:42.11 0 ExecuteAsync Started
Error: validate-env[0]
   07:05:42.48 0 Error: No LSB modules are available.

Debug: validate-env[0]
   07:05:42.48 0 ExecuteAsync Completed Elapsed:00:00:00.3830639
[2023-12-20T03:05:42.492Z] [INFO] Extenension: Invoking validateEnvironement for: wsl-distro-ubuntu1804OrGreater-default
Debug: validate-env[0]
   07:05:42.54 0 ExecuteAsync Started
Error: validate-env[0]
   07:05:42.92 0 Error: No LSB modules are available.

Information: validate-env[0]
   07:05:42.92 0 The default WSL distribution is Ubuntu 18.04 or greater.
Debug: validate-env[0]
   07:05:42.92 0 ExecuteAsync Completed Elapsed:00:00:00.3942682
```

```
Win|PS> wsl
(base) oliver@osmith-pc:/mnt/c/Users/oliver.smith$ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.3 LTS"
(base) oliver@osmith-pc:/mnt/c/Users/oliver.smith$ echo $PATH
/home/oliver/miniconda3/bin:/home/oliver/miniconda3/condabin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/lib/wsl/lib:/mnt/c/Program Files/PowerShell/7:/mnt/c/Python310/Scripts:/mnt/c/Python310:/mnt/c/Program Files/Microsoft/jdk-11.0.16.101-hotspot/bin:/mnt/c/WINDOWS/system32:/mnt/c/WINDOWS:/mnt/c/WINDOWS/System32/Wbem:/mnt/c/WINDOWS/System32/WindowsPowerShell/v1.0:/mnt/c/WINDOWS/System32/OpenSSH:/mnt/c/ProgramData/chocolatey/bin:/mnt/c/Program Files/dotnet:/mnt/c/Program Files/Perforce:/mnt/c/Program Files/Microsoft VS Code/bin:/mnt/c/Program Files/PowerShell/7:/mnt/c/Program Files (x86)/NVIDIA Corporation/PhysX/Common:/mnt/c/Program Files/NVIDIA Corporation/NVIDIA NvDLISR:/mnt/c/WINDOWS/system32:/mnt/c/WINDOWS:/mnt/c/WINDOWS/System32/Wbem:/mnt/c/WINDOWS/System32/WindowsPowerShell/v1.0:/mnt/c/WINDOWS/System32/OpenSSH:/mnt/c/Program Files/Microsoft SQL Server/150/Tools/Binn:/mnt/c/Program Files/Git/cmd:/mnt/c/Program Files/dotnet:/Docker/host/bin:/mnt/c/Program Files/Go/bin:/mnt/c/Program Files/Perforce:/mnt/c/Program Files (x86)/Windows Kits/10/Windows Performance Toolkit:/mnt/c/Program Files/Microsoft SQL Server/130/Tools/Binn:/mnt/c/Program Files/Microsoft SQL Server/Client SDK/ODBC/170/Tools/Binn:/mnt/c/Program Files/nodejs:/mnt/c/Users/oliver.smith/AppData/Local/Microsoft/WindowsApps:/mnt/c/Program Files/Neovim/bin:/mnt/c/Users/oliver.smith/AppData/Roaming/Python/Python310/Scripts:/mnt/c/Users/oliver.smith/.cargo/bin:/snap/bin
(base) oliver@osmith-pc:/mnt/c/Users/oliver.smith$ conda --version
conda 23.11.0
```

**Comment by CberYellowstone** [comment]
This hard-coded design is outrageous. What a shitty code.

**Comment by acube3** [comment]
Thank you for reporting. We will investigate and try to have a fix in a future release.

**Comment by ningx-ms** [comment]
@kfsone We just released 0.2.2 preview, please give it a try and let us know if the Conda installation issue has been resolved. Thx.
